index.html

```diff
@@ -114,6 +114,37 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Please rename this 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` into 
`<!-- ________ your_name Contributor card START ________ -->`